mock/el8.cfg

```diff
@@ -75,15 +75,15 @@ gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Officia
 
 [foreman]
 name=foreman
-baseurl=https://yum.theforeman.org/nightly/el8/x86_64/
+baseurl=https://yum.theforeman.org/3.13/el8/x86_64/
 
 [foreman-plugins]
 name=foreman-plugins
-baseurl=https://yum.theforeman.org/plugins/nightly/el8/x86_64/
+baseurl=https://yum.theforeman.org/plugins/3.13/el8/x86_64/
 
 [katello]
 name=katello
-baseurl=https://yum.theforeman.org/katello/nightly/katello/el8/x86_64/
+baseurl=https://yum.theforeman.org/katello/4.15/katello/el8/x86_64/
```

Comment:
We should have really dropped these in nightly already because they should no longer exist. I know @Odilhao questioned its use, but I didn't dive deep enough to take a look.